### DISCUSSION TITLE
Sorry - no notes for Summer of C# sessions

### DISCUSSION THREAD

**Discussion by MadsTorgersen** [dotnet/csharplang#6419] [LDM Notes]
Hi all,

The [C# LDM agenda](https://github.com/dotnet/csharplang/tree/main/meetings/2022#upcoming-meetings-for-2022) has been showing a number of "Summer of C#" sessions. We did indeed have four broader internal meetings in this series in the normal LDM time slot, but they ended up being of an explorative nature that wasn't easily captured to notes, and in particular they deliberately did not involve design decisions. As a result, I will be deleting them from the agenda without notes being posted for them.

The insights that were gained will filter into existing and new proposals, as well as regular design notes as we start addressing them in LDM. Still, I understand that some people will be disappointed that no artifacts came out from this. This series was the first we've had of its kind, and somewhat experimental. If we have a similar one next year, we will think about the issue of dissemination ahead of time based on the learnings from this summer.

Thanks,

Mads

**Comment by PathogenDavid** [comment]
Disappointing but understandable, thanks for the transparency!

**Comment by huoyaoyuan** [comment]
I'd like to see some high-level ideas, in supplement of detailed design notes and triaging for existing proposals.

For example, in which domains would you value most? What's expected for planning between major evolutions and smallish features? What steps are expected for large meta-features like more generic math and HKT?

These don't need to be concrete decisions or plans. I just want to see the explorations.
